src/ts/ConvivaAnalytics.ts

```diff
@@ -423,6 +446,20 @@ export class ConvivaAnalytics {
     this.client.updateContentMetadata(this.sessionKey, this.contentMetadataBuilder.build());
   }
 
+  // This will propagate content metadata overrides to the conviva instance of the receiver app
+  private propagateOverridesToReceiver(metadataOverrides?: Metadata) {
+    metadataOverrides = metadataOverrides ? metadataOverrides : this.contentMetadataBuilder.getOverrides();
```

Comment:
I'd prefer a classic logical OR.
```suggestion
    metadataOverrides = metadataOverrides || this.contentMetadataBuilder.getOverrides();
```